internal/service.go

```diff
@@ -216,21 +220,24 @@ func (h *HauserService) ProcessFilesByDay(exports []client.ExportMeta) (int, err
 	return len(processedBundles), nil
 }
 
-func (h *HauserService) LoadBundles(filename string, bundles ...client.ExportMeta) error {
-	var objPath string
-	var err error
-	if objPath, err = h.warehouse.UploadFile(filename); err != nil {
-		log.Printf("%s", h.warehouse.GetUploadFailedMsg(filename, err))
+func (h *HauserService) LoadBundles(ctx context.Context, filename string, bundles ...client.ExportMeta) error {
```

Comment:
I think it would be ideal if this took a reader instead of a filename, but I the changes were already pretty large, so I decided to hold off.

Also, we could probably rework this to be separate steps, one where we "stream" data to `SaveFile` and one where we load the data into the database and delete the file in storage.